docset/winserver2022-ps/netwnv/Get-NetVirtualizationProviderAddress.md

```diff
@@ -23,48 +23,55 @@ Get-NetVirtualizationProviderAddress [-ProviderAddress <String[]>] [-InterfaceIn
 ```
 
 ## DESCRIPTION
-The **Get-NetVirtualizationProviderAddress** cmdlet gets Provider Addresses configured in Microsoft® Hyper-V® Server 2016 Network Virtualization.
-A Provider Address is an IPv4 or IPv6 address that Network Virtualization uses for multiple virtual Customer Addresses.
-For more information, see [Network Virtualization technical details](https://technet.microsoft.com/library/jj134174.aspx) on TechNet.
 
-You can specify which Provider Addresses to get by using address state, interface index, prefix length, IP address, or VLAN ID.
-You can use this cmdlet to get Provider Addresses for other cmdlets, such as the Set-NetVirtualizationProviderAddress cmdlet or the Remove-NetVirtualizationProviderAddress cmdlet.
+The `Get-NetVirtualizationProviderAddress` cmdlet gets Provider Addresses configured in Microsoft®
+Hyper-V® Server 2016 Network Virtualization. A Provider Address is an IPv4 or IPv6 address that
+Network Virtualization uses for multiple virtual Customer Addresses. For more information, see
+[Network Virtualization technical details](https://technet.microsoft.com/library/jj134174.aspx) on
+TechNet.
+
+You can specify which Provider Addresses to get by using address state, interface index, prefix
+length, IP address, or VLAN ID. You can use this cmdlet to get Provider Addresses for other cmdlets,
+such as the `Set-NetVirtualizationProviderAddress` cmdlet or the
+`Remove-NetVirtualizationProviderAddress` cmdlet.
 
 ## EXAMPLES
 
 ### Example 1: Get Provider Addresses for an interface
-```
-PS C:\> Get-NetVirtualizationProviderAddress -InterfaceIndex 13
+
+```powershell
+Get-NetVirtualizationProviderAddress -InterfaceIndex 13
 ```
 
 This command gets the Provider Address for the interface with the specified index.
 
 ### Example 2: Get a specific Provider Address
-```
-PS C:\>Get-NetVirtualizationProviderAddress -ProviderAddress "172.16.2.3"
+
+```powershell
+Get-NetVirtualizationProviderAddress -ProviderAddress '172.16.2.3'
 ```
 
 This command gets the Provider Address that has the IP address 172.16.2.3.
 
 ## PARAMETERS
 
 ### -AddressState
-Specifies an array of states of Provider Addresses.
-The acceptable values for this parameter are:
+
+Specifies an array of states of Provider Addresses. The acceptable values for this parameter are:
 
 - Preferred.
```

Comment:
I can't suggest this as an edit to the section, but if you can make this change, that would be great! It'll make things a little more readable for folks and clarify that these values are specific enums:

```markdown
- `Preferred` - The address is unique.
- `Tentative` - The address is not yet verified.
- `Duplicate` - There is a duplicate address on the network.
- `Invalid` - The address lifetime has expired.
- `Deprecated` - The address cannot start new connections.
```